src/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -126,7 +126,7 @@ export const ConfirmBidRoute: React.FC<BidProps> = props => {
         <Separator />
         <BidForm
           initialSelectedBid={getInitialSelectedBid(props.location)}
-          showPricingTransparency={Boolean(/pt=1/.test(props.location.search))}
+          showPricingTransparency={false}
```

Comment:
Looks like this is the default value but I imagine you're wanting to be explicit here?

Reply:
Yeah, for now that was my thought.